leases/utils.py

```diff
@@ -1,11 +1,19 @@
 from datetime import date
 
 
-# If a lease object is being created before 10.6, then the dates are in the same year.
-# If the object is being created between those dates, then the start date is
-# the date of creation and end date is 14.9 of the same year.
-# If the object is being created after 14.9, then the dates are from next year.
-def calculate_berth_lease_start_date():
+def calculate_berth_lease_start_date(use_default_date: bool = False) -> date:
```

Comment:
Could we have two separate functions: `calculate_season_start_date` and `calculate_berth_lease_start_date`? Then there would be no need for this flag param.

Reply:
:hm: yeah, that could be a cleaner approach 🤔

Reply:
I like @santtul 's suggestions :simple_smile: